libcxx/test/std/numerics/numeric.ops/numeric.ops.gcd/gcd.pass.cpp

```diff
@@ -143,5 +162,10 @@ int main(int argc, char**)
     assert(res == 2);
     }
 
-  return 0;
+    do_fuzzy_tests<std::uint8_t>();
+    do_fuzzy_tests<std::uint16_t>();
+    do_fuzzy_tests<std::uint32_t>();
+    do_fuzzy_tests<std::uint64_t>();
```

Comment:
Because [`std::gcd` works for negative numbers](https://godbolt.org/z/dfEbdEWzW), we should test also signed types.